### PR TITLE
Basic Metrics Test

### PR DESCRIPTION
**Pull request by pavelzag**
Purpose or Intent
=================

This is a basic test that checks if Metrics for OSE are up
{{pytest: cfme/tests/containers/test_basic_metrics.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 44f941940a2e566432cbc7392594909885652273:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by pavelzag** [comment]
@dajoRH merge please

**Comment by pavelzag** [comment]
@dajoRH please merge

**Comment by dajoRH** [comment]
Response to comment #246320397:

It seems like all the basic requirements were met, I have therefore merged your PR.

File: cfme/tests/containers/test_basic_metrics.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def test_basic_metrics(provider):
+    """ Basic Metrics availability test
+        This test checks that the Metrics service is up
+        Opens the hawkular status page and checks if it's up
+        """
+    hostname = 'https://' + conf.cfme_data.get('management_systems', {})[provider.key]\
+        .get('hostname', []) + '/hawkular/metrics'
+    ensure_browser_open()
+    sel.get(hostname)
+    element = sel.ByText('STARTED')
```
Comment (mfalesni) [inline cfme/tests/containers/test_basic_metrics.py]: This is completely wrong. `sel.ByText` is just a wrapper object that is used when feeding a value to select. So this object **is not** an element and will **always** return true.

You might like to do something like:

```python
assert sel.elements('//*[contains(., "STARTED")]')
```

Which makes much more sense.

The test you wrote always succeeds if it goes past line 23.

**Comment by pavelzag** [comment]
@mfalesni I changed it here:
https://github.com/ManageIQ/integration_tests/pull/3414

**Pull request merged**